### PR TITLE
Fix permissions for entrypoint_nats.sh

### PR DESCRIPTION
**Pull request by johan-juntunen**

### CODE REVIEW

**Review by pentestiing — APPROVED** [review]
:video_game:

**Review by andtokm — APPROVED** [review]

File: modules/sc-mesh-secure-deployment/src/nats/initd/start_alfred_upper.sh
Comment (pentestiing) [inline modules/sc-mesh-secure-deployment/src/nats/initd/start_alfred_upper.sh:1]: There are other scripts with similar names under `.../src/2_0/...`, plz also update their permissions :lock_with_ink_pen:

**Review by pentestiing — CHANGES_REQUESTED** [review]
:lobster:

File: modules/utils/docker/entrypoint_nats.sh
Comment (TIISR) [inline modules/utils/docker/entrypoint_nats.sh:1]: Is it normal that this file is empty? :)

**Review by TIISR — CHANGES_REQUESTED** [review]

File: modules/utils/docker/entrypoint_nats.sh
Comment (johan-juntunen) [reply modules/utils/docker/entrypoint_nats.sh:1]: There is content in the file, Github just show file rights change

File: modules/sc-mesh-secure-deployment/src/nats/initd/start_alfred_upper.sh
Comment (johan-juntunen) [reply modules/sc-mesh-secure-deployment/src/nats/initd/start_alfred_upper.sh:1]: Fixed

**Review by pentestiing — APPROVED** [review]
:sandwich:

**Pull request merged**